cvat-ui/src/components/quality-control/quality-control-page.tsx

```diff
@@ -350,14 +437,7 @@ function QualityControlPage(): JSX.Element {
     }
 
     if (instance) {
-        title = (
-            <Col className='cvat-quality-page-header'>
-                <Title level={4} className='cvat-text-color'>
-                    Quality control for
-                    <Link to={`/tasks/${instance.id}`}>{` Task #${instance.id}`}</Link>
-                </Title>
-            </Col>
-        );
+        title = setupTitle(instance);
```

Comment:
for your information, you may just create anon function and call it immediately

(() => {
    // do some logic
    // return some jsx
})()

Reply:
Used code from merged patch